src/lib/seam/components/AccessCodeForm/TimezonePicker/TimezonePickerListItem.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useRef } from 'react'
+
+import { getTimezoneLabel, getTimezoneOffset } from 'lib/dates.js'
+import { CheckIcon } from 'lib/icons/Check.js'
+
+interface TimezoneListItemProps {
+  value: string
+  isSelected: boolean
+  onClick: () => void
+}
+
+export function TimezoneListItem({
+  value,
+  onClick,
+  isSelected,
+}: TimezoneListItemProps): JSX.Element {
+  const elRef = useRef<HTMLLIElement>(null)
+
+  useEffect(() => {
+    if (!isSelected) {
+      return
+    }
+
+    const { current: item } = elRef
+
+    if (item == null) {
+      return
+    }
+
+    const container = item.closest('.seam-timezones')
```

Comment:
Interesting, can we pass the container as a prop instead of having to search by class name?

Reply:
No longer relevant as using native `select`.